### PR TITLE
I'm looking at this code with reproach

### PR DESCRIPTION
**Pull request by hobinjk**

### CODE REVIEW

**Comment by AprilSylph** [comment]
image

File: Extensions/blacklist.js [hunk 1]
```diff
@@ -943,8 +943,10 @@ XKit.extensions.blacklist = new Object({
 								tmp_word = tmp_word + " " + m_p_words[m_i + 6];
 							}
 
-							tmp_word = tmp_word.replace(/\,/g, '').replace(/\u2026/g, '');
-							tmp_word = tmp_word.replace(/[\.,-\/#!$%\^&\*;:{}=\-_`~()]/g, "").replace(/\s{2,}/g, " ");
+							const thisCodeIsBadAndTheFollowingEscapeIsGarbage = tmp_word;
```
Comment (AprilSylph) [inline Extensions/blacklist.js:946]: maybe a better name would be `unsanitised`

File: Extensions/blacklist.js [hunk 2]
```diff
@@ -955,6 +957,14 @@ XKit.extensions.blacklist = new Object({
 									return m_word;
 								}
 							}
+
+							if (thisCodeIsBadAndTheFollowingEscapeIsGarbage.includes(m_word)) {
```
Comment (AprilSylph) [inline Extensions/blacklist.js:961]: move to L953 for clarity and less repetition
`if (tmp_word.includes(m_word) || unsanitised.includes(m_word) {`

Comment (hobinjk) [reply Extensions/blacklist.js:961]: oh

File: Extensions/blacklist.js (hunk 1 above)
Comment (hobinjk) [reply Extensions/blacklist.js]: done but with the 🇺🇸 spelling 😎

**Pull request merged**